crates/conf/src/plugin.rs

```diff
@@ -60,8 +70,18 @@ fn poll_conf(
                     toasts.send(ToastEvent::new(format!(
                         "Configuration loading failed: {err}"
                     )));
-                    let error: &dyn Error = err.as_ref();
-                    log_full_error!(error);
+                    let error: ConfigLoadError = err;
+
+                    match error {
+                        ConfigLoadError::Error(e) => {
+                            error!("Config loading failed: {:?}", e)
+                        }
+                        ConfigLoadError::CheckErrors(e) => {
+                            for err in e {
```

Comment:
I would improve the log by giving it a header + unifying multiple logs into a single one.

```rust
let mut error_log = String::from("Configuration validation errors:");
for err in errors {
    error_log.push('\n');
    error_log.push_str(&format!("{err}"));
}
```